AllReadyApp/Web-App/AllReady/Views/Shared/_AdminNavigationPartial.cshtml

```diff
@@ -1,31 +1,32 @@
-﻿@using AllReady.Security
+﻿@using AllReady.Areas
+@using AllReady.Security
 
 @if (User.IsUserType(UserType.SiteAdmin) || User.IsUserType(UserType.OrgAdmin))
 {
     var descriptor = (Microsoft.AspNetCore.Mvc.Controllers.ControllerActionDescriptor)ViewContext.ActionDescriptor;
     var areaName = descriptor.RouteValues["area"]; // there must be a better way to get the area name!
     var controllerName = descriptor.ControllerName;
-    var isAdmin = areaName == "Admin";
+    var isAdmin = areaName == AreaNameConstants.ADMIN_AREA_NAME;
     <li class="dropdown dropdown-admin @(isAdmin? "active":"")">
         <a class="dropdown-toggle" data-toggle="dropdown" href="#" role="button" aria-haspopup="true" aria-expanded="false">
             Admin <span class="caret"></span>
         </a>
         <ul class="dropdown-menu">
             @if (User.IsUserType(UserType.SiteAdmin))
             {
-                <li @(isAdmin && controllerName == "Organization" ? " class=active" : "")><a asp-area="Admin" asp-controller="Organization" asp-action="Index">Organizations</a></li>
+                <li @(isAdmin && controllerName == "Organization" ? " class=active" : "")><a asp-area=@AreaNameConstants.ADMIN_AREA_NAME asp-controller="Organization" asp-action="Index">Organizations</a></li>
```

Comment:
thanks, I'll fix all those up, it tested fine but I'm not as familiar with these types of pages.